assets/js/common/mozilla-utils.js

```diff
@@ -20,7 +20,7 @@ if (typeof Mozilla === 'undefined') {
      * @param {userAgent} optional UA string for testing purposes.
      */
 
-    Utils.triggerIEDownload = function(link, userAgent) {
+    Utils.triggerIEDownload = function (link, userAgent) {
```

Comment:
Spaces before function parens is inconsistent with all the other JS in this repository, so all these lines shouldn't be here. Making style changes in adjacent code like this also just adds a lot of noise to PRs.